apps/i18n/common/en_us.json

```diff
@@ -578,7 +578,13 @@
   "deleteAccountDialog_emailUs": "If you delete your account and change your mind, you can email us at support@code.org within 3 weeks to recover your account.",
   "deleteAccountDialog_verification": "To verify, type {verificationString} below:",
   "deleteAccountDialog_verificationString": "DELETE MY ACCOUNT",
+  "deleteAnimation": "Delete animation",
+  "deleteAnimationConfirm": "Are you sure you want to delete this animation? You cannot undo this action.",
+  "deleteBackground": "Delete background",
```

Comment:
These strings are unused? Only the animation ones are being used right now.

Reply:
Yes, the costume and background strings are currently unused. We'll want the costume strings for the next update here since this modal is used in both Game Lab and Sprite Lab. I thought it made sense to add them as soon as possible so they'd be most likely to be translated ahead of HoC. The background strings likely wouldn't be needed until we rework everything for collections, so if it's a better practice to pull them out until needed, I'm happy to do so.

Reply:
I don't know the answer to this but maybe @madelynkasula or @epeach will- can we use a parameterized string here and still have the words "costume", "animation", and "background" get translated?

Reply:
yeah, you can -- you'd have to nest translated strings, which i usually prefer over in this situation. the outer string would accept a parameter, which can also be a translated string

Reply:
@ajpal Are there changes you would request here?

Reply:
Yeah, let's make the string parameterized to reduce duplication. Here's an example of a parameterized string: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/i18n/common/en_us.json#L288 and here's how it gets called: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/storage/GetColumnParamPicker.jsx#L54

And then as Maddie suggested, you'd just add the strings "costume", "animation", and "background" separately, and use that as the parameter